cli/app.go

```diff
@@ -1103,6 +1109,65 @@ Example:
 							},
 							Action: ModuleBuildLocalAction,
 						},
+						{
+							Name:        "start",
+							Description: "start a remote build",
+							Flags: []cli.Flag{
+								// todo: factor out common flags
+								&cli.StringFlag{
+									Name:      moduleFlagPath,
+									Usage:     "path to meta.json",
+									Value:     "./meta.json",
+									TakesFile: true,
+								},
+								&cli.StringFlag{
+									Name:     moduleBuildFlagVersion,
+									Usage:    "version of the module to upload (semver2.0) ex: \"0.1.0\"",
+									Required: true,
+								},
+							},
+							Action: ModuleBuildStartAction,
+						},
+						{
+							Name:  "list",
+							Usage: "check on the status of your cloud builds",
+							Flags: []cli.Flag{
+								&cli.StringFlag{
+									Name:      moduleFlagPath,
+									Usage:     "path to meta.json",
+									Value:     "./meta.json",
+									TakesFile: true,
+								},
+								&cli.IntFlag{
+									Name:        moduleBuildFlagNumber,
+									Usage:       "number of builds to list",
+									Aliases:     []string{"n"},
+									DefaultText: "all",
```

Comment:
I tried to look this up myself but couldn't figure it out. Should `DefaultText` always come with a `Value`? Is it that `DefaultText` is a suggested value where as `Value` is an actual default?

Reply:
Yup, this surprising behavior was fixed for the other cli commands in https://github.com/viamrobotics/rdk/pull/3162

My understanding is that:
`DefaultText` => shows in help msg, does not actually change the default value when you try to read the flag
 `Value` => shows in help msg + actual default value

Reply:
Yup thanks for confirming what I found too. I feel like `DefaultText` isn't appropriate then. The "default" is that you don't specify the flag at all, otherwise you specify it and it should be a int.

Reply:
Ok, as we discussed keep it as-is.